src/pages/RequisitionsPage.js

```diff
@@ -31,7 +31,8 @@ export class RequisitionsPage extends React.Component {
       isRequestView: true,
       isCreatingRequisition: false,
     };
-    this.requisitions = props.database.objects('Requisition');
+    this.requisitions = props.database.objects('Requisition')
+                                      .filtered('serialNumber != "-1"');
```

Comment:
Quick and correct IMO. Code smells include:
- A bug happened (start sniffing...)
- The 'post sync' processor, which has nothing to do with UI, is using something called a 'UIDatabase' (there it is!)